com.stencyl.Game.json

```diff
@@ -20,6 +20,10 @@
             "sources": [
                 {
                     "type": "file",
+                    "path": "com.stencyl.Game.png"
```

Comment:
Because we name the icon `com.stencyl.Game.png` in the desktop file, flatpak expects that icon to exported as well.